### PR TITLE
MORE 3D OBJECTS

### PR DESCRIPTION
**Pull request by DLu**
Since figuring out the process with PointClouds, I went ahead and added a bunch more 3d objects that have direct analogues in rviz, including
 * Path
 * LaserScan
 * Polygon
 * Pose
 * Pose Array
 * Odometry
 * PointStamped

Here's a video: http://youtu.be/U8bBus27kSg

### CODE REVIEW

**Comment by rbonghi** [comment]
:+1:

File: src/navigation/Odometry.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+  this.sns = [];
+
+  var rosTopic = new ROSLIB.Topic({
+      ros : ros,
+      name : topic,
+      messageType : 'nav_msgs/Odometry'
+  });
+
+    rosTopic.subscribe(function(message) {
```
Comment (T045T) [inline src/navigation/Odometry.js]: Looks like the indentation is off here

**Comment by DLu** [comment]
Updated                   the                

                       white                                                             space.

**Comment by rctoris** [comment]
:+1: awesome!!!

**Pull request merged**